src/common/base/base.scss

```diff
@@ -34,8 +34,13 @@ a:hover{
     color: $brand-primary;
 }
 
-h3, h4{
-    font-size: 1em;
+h3{
```

Comment:
@nickdotreid @martokarski I believe styles for h3 and h4 should also have a prefix so they apply only in settings as they are causing a bit of a mess in other places in UI